blueprints/hr_assistant/webex_bot_server.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+
+import os
+import logging
+import json
+from flask import Flask, request
+import requests
+from ciscosparkapi import CiscoSparkAPI
+
+from mindmeld.components import NaturalLanguageProcessor
+from mindmeld.components.dialogue import Conversation
+from mindmeld import configure_logs
+
+class WebexBotServer():
+
+    def __init__(self):
+        self.app = Flask(__name__)
+        self.logger = logging.getLogger(__name__)
+
+        # Create web hook here: https://developer.webex.com/docs/api/v1/webhooks/create-a-webhook
+        self.WEBHOOK_ID = os.environ.get('WEBHOOK_ID')
+
+        # Create bot access token here: https://developer.webex.com/my-apps/new
+        self.ACCESS_TOKEN = os.environ.get('BOT_ACCESS_TOKEN')
+
+        if not self.WEBHOOK_ID or not self.ACCESS_TOKEN:
+            raise Exception('WEBHOOK_ID and BOT_ACCESS_TOKEN are not set')
+
+        self.spark_api = CiscoSparkAPI(self.ACCESS_TOKEN)
+        self.ACCESS_TOKEN_WITH_BEARER = 'Bearer ' + self.ACCESS_TOKEN
+        self.CISCO_API_URL = 'https://api.ciscospark.com/v1'
+
+        configure_logs()
+        self.nlp = NaturalLanguageProcessor('.')
+        self.nlp.build()
+        self.conv = Conversation(nlp=self.nlp, app_path='.')
+
+
+    def run(self, host='localhost', port=port_number):
```

Comment:
port_number is not a constant

```suggestion
    def run(self, host='localhost', port=7150):
```